frontends/ast/simplify.cc

```diff
@@ -145,7 +145,7 @@ void AstNode::fixup_hierarchy_flags(bool force_descend)
 
 // Process a format string and arguments for $display, $write, $sprintf, etc
 
-Fmt AstNode::processFormat(int stage, bool sformat_like, int default_base, size_t first_arg_at) {
+Fmt AstNode::processFormat(int stage, bool sformat_like, int default_base, size_t first_arg_at, bool required) {
```

Comment:
Please rename this parameter to `may_fail`. (The function would be inverted in this case.)

Reply:
Should I edit the commit, or add a new one?

Reply:
I prefer editing the commit.

Reply:
Done.